examples/certification/certified-counter/src/frontend/index.html

```diff
@@ -5,13 +5,13 @@
     <meta name="viewport" content="width=device-width" />
```

Comment:
This project was moved from Webpack to Vite to be consistent with the other example projects. Vite is also much simple to configure compared to Webpack. So there are some minor code changes here to work with Vite.